### PR TITLE
add Wi2Connect for Mac

### PR DESCRIPTION
**Pull request by aehata**
Add a new Cask. :)
- Wi2 Connect for Mac

### CODE REVIEW

File: Casks/wi2-connect.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'wi2-connect' do
+  version :latest
+  sha256 '2946fb1c6997d64430eb7c8f95d37262df1c1add636a6c3571e36735ee76d0e5'
+
+  url 'https://service.wi2.ne.jp/sw/autoLogin/MacOSX/Wi2-Connect-MacOSX.dmg'
+  #name ''
```
Comment (vitorgalvao) [inline Casks/wi2-connect.rb]: This needs to be set.

**Comment by aehata** [comment]
Thanks for the review.
I set name stanza.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 29dbd0652df7c734180b1b8ddda38a31d1f57b87. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**